node_modules/gh-series/tests/test-events.js

```diff
@@ -494,5 +494,47 @@ describe('Event series', function() {
                 });
             });
         });
+
+        /**
+         * Test that verifies that organisers their personal information is hidden
+         */
+        it('verify organisers their personal information is hidden', function(callback) {
+            TestsUtil.generateTestUsers(global.tests.apps.cam2013, 2, false, function(simon, nico) {
+
+                // Generate a test serie with some events
+                var calendarStart = moment().subtract(1, 'day').format();
+                var calendarEnd = moment().add(30, 'day').format();
+                SeriesTestUtil.generateSerieWithEvents(simon.client, 1, 10, calendarStart, calendarEnd, function(series) {
+                    var serie = series[0];
+                    var event = serie.events[0];
+
+                    // Add Nico as an organiser
+                    var update = {};
+                    update[nico.profile.id] = true;
+                    EventsTestUtil.assertUpdateEventOrganisers(simon.client, event.id, update, function() {
+
+                        // Get the series events
+                        SeriesTestUtil.assertGetSeriesEvents(simon.client, serie.id, null, null, null, null, function(events) {
+
+                            // Assert nico's personal information is hidden
+                            var seenNico = false;
+                            _.each(events.results, function(event) {
```

Comment:
Most of the organisers are plain-text authors here, there's nothing to index them on